### PR TITLE
Created yaml-files

### PR DESCRIPTION
**Pull request by shardi-b**
1. Created a yaml for each example. 
2. Make sure each yaml file has minimal settings required for running. 
3. Updated each example *.py file to include how to run with yaml file

### CODE REVIEW

File: examples/IMAGE/cifar/cifar.py [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
 This file demonstrates a training pipeline for an CIFAR Neuron.
+
 Example:
-        $ python examples/cifar.py
+        $ python examples/IMAGE/cifar/cifar.py
+
+Look at the yaml config file to tweak the parameters of the model. To run with those 
+default configurations, run:
+        $ cd examples/IMAGE
+        $ python cifar/cifar.py --session.config_file cifar/cifar_config.yaml
```
Comment (shibshib) [inline examples/IMAGE/cifar/cifar.py]: Nice documentation!

File: examples/IMAGE/cifar/cifar_config.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+#   resid_pdrop: 0.1
+#   embd_pdrop: 0.1
+#   attn_pdrop: 0.1
+#   layer_norm_epsilon: 1.0e-05
+#   summary_type: cls_index
+#   initializer_range: 0.02
+#   summary_activation: null
+#   summary_proj_to_labels: true
+#   summary_first_dropout: 0.1
+#   summary_use_proj: true
```
Comment (shibshib) [inline examples/IMAGE/cifar/cifar_config.yaml]: There are args for cifar model, take a look at `bittensor/synapses/dpn.py` we should add those here. Better than commented out yaml elements

File: examples/IMAGE/ffnn_grunt/ffnn_grunt_config.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+# Make sure the path of your coldkeyfile and hotkeyfile match!
+neuron:
+  coldkeyfile: ~/.bittensor/wallets/default/coldkeypub.txt
+  hotkeyfile: ~/.bittensor/wallets/default/hotkeys/default
+synapse:
+  target_dim: 10
+
+# Currently do not have any configs for the dendrite but could use in the future:
+# dendrite:
```
Comment (shibshib) [inline examples/IMAGE/ffnn_grunt/ffnn_grunt_config.yaml]: No need for this one, we can add that blurb in the documentation

File: examples/IMAGE/mnist/mnist_config.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  batch_size_train: 8
+  batch_size_test: 64
+  log_interval: 10
+  sync_interval: 10
+  name: mnist
+  record_log: true
+nucleus:
+  max_workers: 5
+  queue_maxsize: 10000
+  queue_timeout: 5
```
Comment (shibshib) [inline examples/IMAGE/mnist/mnist_config.yaml]: same comment as CIFAR, take a look at `bittensor/synapses/ffnn.py` and add those args under `synapse:`

File: requirements.txt [hunk 5]
```diff
@@ -26,7 +26,6 @@ pycrypto==2.6.1
 py-sr25519-bindings==0.1.2
 py-ed25519-bindings==0.1.1
 py-bip39-bindings>=0.1.6
-pytest-asyncio
```
Comment (shibshib) [inline requirements.txt]: How come we took out pytest-asyncio? Any reason?

**Review by shibshib — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 5 above)
Comment (shardi-b) [reply requirements.txt]: it was a duplicate requirement!

**Review by unconst — APPROVED** [review]

File: examples/IMAGE/cifar/cifar_config.yaml [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  sync_interval: 10
+  name: cifar
+  record_log: true
+nucleus:
+  max_workers: 5
+  queue_maxsize: 10000
+  queue_timeout: 5
+neuron: # Make sure your coldkeyfile and hotkeyfile locations are correct!
+  coldkeyfile: ~/.bittensor/wallets/default/coldkeypub.txt 
+  hotkeyfile: ~/.bittensor/wallets/default/hotkeys/default
```
Comment (shibshib) [inline examples/IMAGE/cifar/cifar_config.yaml]: just noticed. I think that needs to be `wallet:` not neuron :)

**Review by shibshib — APPROVED** [review]

**Pull request merged**